sdk/go/x/auto/stack.go

```diff
@@ -159,6 +159,25 @@ func SelectStack(ctx context.Context, fqsn string, ws Workspace) (Stack, error)
 	return s, nil
 }
 
+// UpsertStack creates a new stack using the given workspace and fully
+// qualified stack name (org/project/name) if the stack does not already exist.
```

Comment:
It would be nice to clarify that UpsertStack tries to create a new stack, but falls back to selecting an existing stack if one already exists.